src/tests/testACLMaxUserIP.cc

```diff
@@ -56,10 +53,21 @@ TestACLMaxUserIP::testDefaults()
     CPPUNIT_ASSERT_EQUAL(false,anACL.valid());
 }
 
+/// customizes our test setup
+class MyProgram: public TestProgram
+{
+public:
+    virtual ~MyProgram() = default;
+
+    /* TestProgram API */
+    void startup() override;
+};
+
+RegisterTestProgram(MyProgram);
+
 void
-TestACLMaxUserIP::setUp()
+MyProgram::startup()
 {
-    CPPUNIT_NS::TestFixture::setUp();
     Acl::RegisterMaker("max_user_ip", [](Acl::TypeName name)->ACL* { return new ACLMaxUserIP(name); });
```

Comment:
This was probably a bug waiting to be discovered -- we were registering the same ACL maker function twice, once for each TestACLMaxUserIP test case.